prime-router/metadata/hl7_mapping/ORU_R01/base/xad-extended-address.yml

```diff
@@ -1,36 +1,79 @@
 name: ORU-R01-xad-extended-address
+constants:
+  # if orderIndex is provided, we use the "-ordered" elements to support multiple entries
+  fieldPath: (%{hl7AddressField})(%{orderIndex})
 elements:
   - name: street-address
     condition: '%resource.line.count() > 0'
     value: [ '%resource.line[0]' ]
     hl7Spec: [ '%{hl7AddressField}-1' ]
 
+  - name: street-address-ordered
+    condition: '%{orderIndex} NOT_NULL && %resource.line.count() > 0'
+    value: [ '%resource.line[0]' ]
+    hl7Spec: [ '%{fieldPath}-1' ]
```

Comment:
This is confusing having fieldPath and hl7AddressField

Reply:
fixec